### ISSUE TITLE
balanceData error: Please make sure your system time is synchronized, or pass the useServerTime option.

### ISSUE THREAD

**Issue by charlesastwood** [jaggedsoft/php-binance-api#39]
Trying to use this in Symfony 4. Works perfectly through a controller and displaying results on a page. However if I try to use the `balances` in a command when running the cli command I get:

```
signedRequest error: {"code":-1022,"msg":"Signature for this request is not valid."}
balanceData error: Please make sure your system time is synchronized, or pass the useServerTime option.
```

Saw this ticket:
https://github.com/jaggedsoft/php-binance-api/issues/35

I think I need to use useServerTime but it isn't implemented yet?

I echoed out the $endpoint from the call and it looks like (signature modified for obscurification):

`https://api.binance.com/api/v3/account?timestamp=1516900612521&signature=df5a4338fa3421f8ff79afbdbe3df1cb8c87a288512c9e648f6260ee75114446`

**Comment by jaggedsoft** [comment]
It's not implemented yet, I'll get it done as soon as I can. Though, the error you're getting, "signature for this request is not valid," has nothing to do with useServerTime, I will need to update that error message

Can you make any other signed request calls? It seems to think your API key/secret is incorrect

**Comment by charlesastwood** [comment]
Is very strange as it works on the web site, it's just when it goes to CLI that it has a problem. I'll continue to try and debug the calls to see if there is any difference between the URL calls between the web version and CLI but I can't see how as they both use the same service

**Comment by jaggedsoft** [comment]
I use this 100% CLI only, wish I could help more. I'm wondering if there is some php.ini setting difference between your two environments that's causing this. If so I'm curious to know what it is

**Comment by jaggedsoft** [comment]
Maybe there is a problem with getting this to work with Symfony. I'll do more research on this when I have time

**Comment by jaggedsoft** [comment]
Available in `v0.0.12` https://github.com/jaggedsoft/php-binance-api/commit/991578f690a9c91a833e0794779dd3e6aeb78c41
#### useServerTime Method #1
```php
$api = new Binance\API("<key>","<secret>", ['useServerTime'=>true]);
```

#### useServerTime Method #2 
```php
// Call this before running other functions
$api->useServerTime();
```


This is unrelated to the "Signature for this request is not valid" error. However, linking the useServerTime option here in case someone comes here looking for that
